cad/logisim-evolution/Portfile

```
@@ -23,7 +23,7 @@ checksums           rmd160  87b24defcabd6ab81333bc1c52c76928a64bdab1 \
                     size    40050282
 
 java.version        9+
```

Does this need to be updated as well?

```suggestion
java.version        14+
```

---

That's a good spot. Thanks @chrstphrchvz.